packages/openapi-generator/src/generator.ts

```diff
@@ -353,7 +351,10 @@ async function generatePackageJson(
     'package.json',
     packageJson({
       npmPackageName: packageName,
-      description: genericDescription(directoryName),
+      description: packageDescription(
+        directoryName,
+        (directoryNameRaw: string) => directoryNameRaw.split('-').join(' ')
+      ),
```

Comment:
[nth] Seems like duplicate code to me. With my other comments, I guess this can be made obsolete?